### DISCUSSION TITLE
Token to send to my gateway

### DISCUSSION THREAD

**Discussion by williamtdepaula** [hyochan/react-native-iap#1346] [Q&A]
Hello, I would like to know if with this lib it is possible for me to obtain the authorization token to send to my gateway that will perform the payment, I am using Adyen as a gateway.

**Comment by andresesfm** [comment]
No, that gateway is for physical goods. Whereas this one is for in-app purchases

**Reply by IgorDePaula** [reply]
What's difference? The payment is digital. Can I use any gateway for this? I not found about this in docs @andresesfm

**Reply by andresesfm** [reply]
This library is for mobile-native IAP. Which are meant for digital goods. The stores take a large percentage of the transaction.

**Reply by IgorDePaula** [reply]
Ok, what stores? I'm look for a gateway that make split payment in many currencies. Any suggestion?
